### PR TITLE
Create swedish.json

### PR DESCRIPTION
**Pull request by XCOM-HUB**
I have finally translated swedish.json 🥳

### CODE REVIEW

File: Mods/vcmi/config/vcmi/swedish.json [hunk 1]
```diff
@@ -0,0 +1,672 @@
...
+	"vcmi.systemOptions.scalingButton.hover"               : "Gränssnittsskalning: %p%",
+	"vcmi.systemOptions.scalingButton.help"                : "{Gränssnittsskalning}\n\nÄndrar storleken av de olika gränssnitten som finns i spelet.",
+	"vcmi.systemOptions.scalingMenu.hover"                 : "Välj gränssnittsskalning",
+	"vcmi.systemOptions.scalingMenu.help"                  : "Förstorar eller förminskar olika gränssnitt i spelet.",
+	"vcmi.systemOptions.longTouchButton.hover"             : "Fördröjt tryckintervall: %d ms", // Översättningsnot: ’ms’ = "millisekunder"
+	"vcmi.systemOptions.longTouchButton.help"              : "{Fördröjt tryckintervall}\n\nNär du använder dig av en pekskärm och vill komma åt spelalternativ behöver du göra en fördröjd pekskärmsberöring under en specifikt angiven tid (i millisekunder) för att en popup-meny skall synas.",
+	"vcmi.systemOptions.longTouchMenu.hover"               : "Välj tidsintervall för fördröjd pekskärmsberöringsmeny",
+	"vcmi.systemOptions.longTouchMenu.help"                : "Ändra varaktighetsintervallet för fördröjd beröring.",
+	"vcmi.systemOptions.longTouchMenu.entry"               : "%d millisekunder","%d millisekunder",
```
Comment (Laserlicht) [inline Mods/vcmi/config/vcmi/swedish.json:215]:
```suggestion
	"vcmi.systemOptions.longTouchMenu.entry"               : "%d millisekunder",
```

**Comment by Laserlicht** [comment]
To enable you also have to edit:
`Mods/vcmi/mod.json`

And add "swedish" in line with the other languages.

File: Mods/vcmi/config/vcmi/swedish.json [hunk 2]
```diff
@@ -0,0 +1,672 @@
...
+	// Strings for HotA Seer Hut / Quest Guards
+	"core.seerhut.quest.heroClass.complete.0"    : "Ah, du är %s.  Här är en gåva till dig.  Tar du emot den?",
+	"core.seerhut.quest.heroClass.complete.1"    : "Ah, du är %s.  Här är en gåva till dig.  Tar du emot den?",
+	"core.seerhut.quest.heroClass.complete.2"    : "Ah, du är %s.  Här är en gåva till dig.  Tar du emot den?",
+	"core.seerhut.quest.heroClass.complete.3"    : "Vakterna ser att du är %s och erbjuder att låta dig passera.  Accepterar du?",
+	"core.seerhut.quest.heroClass.complete.4"    : "Vakterna ser att du är %s och erbjuder att låta dig passera.  Accepterar du?",
+	"core.seerhut.quest.heroClass.complete.5"    : "Vakterna ser att du är %s och erbjuder att låta dig passera.  Accepterar du?",
+	"core.seerhut.quest.heroClass.description.0" : "Skicka %s till %s",
+	"core.seerhut.quest.heroClass.description.1" : "Skicka %s till %s",
+	"core.seerhut.quest.heroClass.description.2" : "Skicka %s till %s", ’Skicka %s till %s’,
```
Comment (Laserlicht) [inline Mods/vcmi/config/vcmi/swedish.json:466]:
```suggestion
	"core.seerhut.quest.heroClass.description.2" : "Skicka %s till %s",
```
another one :D

**Comment by XCOM-HUB** [comment]
> To enable you also have to edit: `Mods/vcmi/mod.json`
> 
> And add "swedish" in line with the other languages.

I have added the necessary code for the Swedish language in mod.json now!

Go check it out #4698

**Pull request merged**